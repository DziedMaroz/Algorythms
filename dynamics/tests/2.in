916144244026976708
405
9
76065520734144
57713780011372
4593049237727999
4989676022153457
2980
0081329866740332
7
3
2035577394
962
949
0
18726536717213
270011922479523
837439559507747
265145513
2
824
58077996
7176442
67
220791324395
54579746330
163
0883050336690147
7182866028863285
46
03145433071
4619
706461476156520
09046
8962861573021251
9745279781185902
5457663770
905862204376409
33
14374671291771650
4
588634944292
19439423412885288
90364
5621
23529970338
974
05863348346398989
756587
03
981826
018009206
9262
417
2512807575
1
6
15
38874050
050030
51623072004
7
197723043973868
508389194
34878678911673928
4957855661148767
21558995843892
30175165
18
56708265523966850
3
2764754375588
61447
46
614460126123
653666660
8973
160950012462226
23645919597063
2326807
885089435979
597
181077595195
349812403

79025580
0
43406059603
576756880315582
8653038086339294
74302
31203
21336134362
901668
637775105925
99
4634
665367963973
8
866994170296873
477310165760973
14087709
3909964603929882
406594661003
870797
325
9410776363

902530229257
86889
9
9310398234266321
07700589379387459
4
863844086388241
0014612132
417600
78665686162694
52049346134092673
37778992
021166
41904141315
73056881945954322
929290121692042
15
9290157130547
0170361
27238887771767791
40735616924
9460783738737
35004205658160220
157551947
1
9633722
4309826623332967
8
7212701725074712

9921700164
37430288539
242
66473
63
4700255611761225
77715119
7078119811
255
004942544689482
539219518587661
76910382648410
231296808591523
47471874
32
715296829400261
9635555127760

57715244
617536937109
330326808072
7873569588
4231725


66860088
34242238717920818
4
810826421054
216
74094
630476566661
36798668266664
4
3981949882950614
49052288
07924
645
810366
621178561394927
1480
797051
6597
1925783816

56766
88508694
5
1400113
077180800
8327
771099
457763500940297
254894
03913088965995053
90
932024470
05003911166506
29
33528630314
943817550016
8844091
3891191518003
692910258440
1933259920466
91
6231981

2038080463676
9433933
7317
0786
036465
4021726078459010
550412896284
871001664
366
37081199
4000
6070325539
070180655326947
8259

913439916892795
6

57248907131
402262
504894
15
18272766361344329
004307511705901
88302049543968286
75448912042434255
92135436641023
16009322681876406
4607471
200
2254315860
885019262708128
7069236327
672401720643
10356098745
32369045014606805
46951
20
6337
82449049
18575284237273656
15
5161939240111342
5
96215965278992
98849495576862
214
9768511292
786980289743436
7687
064469
2537027839
4076
981
7
77964167074239

192035798
2342
481
84634641636
4961328
22984
14062916154
16397055591066
514640852
946874
99492188666126
100
23
843631704334185
6476815856688
30672
2284387
366700
94378123366141
2
289
3394774054742742
550851427964034

1022107
17979929
696263837
94423151590
4
261551603818
0280
909120400645
0416405687675
92708985920418
7118217736845983
599506724
8235807245486
6921870864695
385175
54237259440
434148568986966
950
6
0543694848
09
7602
05413771
4
25079402
9865
5379014536387140
1593822
9685655767
1009973631925572
11
39002349331931
46688
83282930968643288
836700192046876
14408994029077
8995058411333
382684370672
952483043319886
18392
20412

56303682
91871932970
418886
8964798770038726
3207
3944503468733879
41112842420947312
7133477411623099
834182781
063711
091552017190
528536133250953
428
973
3502
0226229652014627
4859
7926076703528126
8
32411415
47148063
4685

6012084323
29713861129894
61090773151321
535429029
29328146650
1
05821382337391650
0791668382
51944643598423
6389020806444543
35545352440
85112293
0927787672854008
227544617380447
75538604341

76774319804304626
3624643209

3428
636
2675682069476887
650
15236162

9
2356465719614962
059211586
43106363296357466
72
59162
852
6
538
05735335293472
00
81185491200771670
9063922925557881
567242027496101
16553603874
351300772361
40753576191895
4438921640403711
176
54031023022881847
13058491
840
191815
809
5353892646

7283434275286982
139562
796340921788347
5693355515385
53710
