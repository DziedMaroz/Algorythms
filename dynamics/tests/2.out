16
91 6 1 4 4 2 4 4 0 2 6 9 7 67 0 8
